## Hi there 👋
👨‍💻 **About Me**

I am a computational biologist and evolutionary genomicist, having recently completed my Ph.D. in Biological Sciences (Evolutionary genomics) at the University of California, Irvine. My research is driven by a fascination with the evolutionary forces that shape genomes. I specialize in leveraging long-read sequencing data and developing bioinformatics pipelines to explore complex and repetitive regions of the genome.

My doctoral thesis, [“Polymorphism in Repetitive Regions: Insights from Long Accurate Sequencing in *Drosophila melanogaster*”](https://escholarship.org/uc/item/4p9322jf), focused on understanding the genetic variation and functional impact of repetitive elements. I am passionate about using code to untangle complex biological questions and am always excited to learn new computational methods.

- 🌱 I’m skilled in using **bash**, **Python** and **R** for large-scale genomic data analysis, pipeline development, and data visualization.
- 💬 Ask me about long-read sequencing, genome assembly, 3D genome architecture or epigenomcis of transposable elements.
- 📫 How to reach me: harsh.g.shukla@gmail.com


🛠 **My Tech Stack & Skills**

My expertise lies at the intersection of biology and computer science. I am proficient with a variety of tools and languages essential for modern genomics research.

🔬 **Research Focus**

My research integrates population genetics, molecular evolution, and functional genomics methods to investigate how genomes evolve. Key areas include:

- De novo Genome Assembly: Assembling high-quality reference genomes using PacBio HiFi and Oxford Nanopore data.
- 3D Genome Architecture: Analyzing Hi-C and epigentic data (Cut&Tag) data to understand how transposable elements influence chromatin structure and gene regulation.
- Population Genomics: Studying patterns of genetic variation, inbreeding, and adaptation in natural populations.

 [My Google scholar profile](https://scholar.google.com/citations?hl=en&user=ONfT_skAAAAJ&view_op=list_works&sortby=pubdate)
